phpunit.xml.dist

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
While running phpunit I am receiving warning `Your XML configuration validates against a deprecated schema.`. Could you fix that?